User-Help/AskingGuidance.md

```diff
@@ -0,0 +1,37 @@
+## How to ask a great question
+
+The best way to get a perfect answer to your questions is to ask a great question. The better your question is, the easier it will be for someone to be able to give you the answer you need. Here are a few things to keep in mind when asking questions.
+
+**Make sure your question is in scope**
+
+Every Codidact site has its own guidelines for what topics are able to be asked about on that site. You wouldn't want to ask about frying eggs on [Writing](https://writing.codidact.com), or bicycle maintainence on [Cooking](https://cooking.codidact.com). See [$SiteName's scope guidelines here](/help/scope).
```

Comment:
Can we link to `/help/faq` for now? We may have a /scope article at some point, but right now the topic guidelines are in the FAQ.